pom.xml

```diff
@@ -127,6 +127,7 @@
         <hive.engine.hive.version>3.1.3</hive.engine.hive.version>
         <hive.version>3.1.3</hive.version>
         <hive.service.rpc.version>3.1.3</hive.service.rpc.version>
+        <hive.storage-api.version>2.7.0</hive.storage-api.version>
```

Comment:
I am not sure we need storage-api for 2.7.0 version, according to *mvn dependency tree*. It seems we have 2.7.2 version jar.

Reply:
1. hive-standalone-metastore:3.1.3 depends on hive-storage-api:2.7.0
2. hive-storage-api:2.7.2 is used in kyuubi-hive-jdbc

So, I think hive-storage-api:2.7.0 is more appropriate.